command/var_get.go

```diff
@@ -0,0 +1,187 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/nomad/api"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+type VarGetCommand struct {
+	Meta
+	outFmt string
+	tmpl   string
+}
+
+func (c *VarGetCommand) Help() string {
+	helpText := `
+Usage: nomad var get [options] <path>
+
+  The 'var get' command is used to get the contents of an existing variable.
+
+  If ACLs are enabled, this command requires a token with the 'var:read'
+  capability.
```

Comment:
I'm just noticing how we've described the ACL requirements here and while I know what this means, it doesn't match up with how the ACL policy is really specified. I'd probably write this out as:

```suggestion
  If ACLs are enabled, this command requires a token with the 'variables:read'
  capability for the variable's namespace.
```

(Similar for the other commands.)